packages/itwinui-css/src/toast/categories.scss

```diff
@@ -18,4 +18,8 @@
   > .iui-toast-anchor {
     @include iui-anchor-status($category);
   }
+
+  > .iui-button {
```

Comment:
Probably should be targeting a class like `.iui-toast-button` instead of `.iui-button`.

Reply:
i think you can just target `button` tbh

Reply:
Ok, I will leave it as is then.

Reply:
just to be clear, i'm suggesting `button` instead of `.iui-button`

Reply:
Ah, I did not pickup on that, change made to target `button`.